src/common/Preforker.h

```diff
@@ -45,6 +46,18 @@ class Preforker {
       return (errno = e, -1);
     }
 
+    struct sigaction sa;
+    memset(&sa, 0, sizeof(sa));
```

Comment:
no need to `memset()` here.

Reply:
Hi, kefu, as I seen in other file, memset is used：https://github.com/ceph/ceph/blob/master/src/global/signal_handler.cc#L54 .  Whether to keep the same code style as before?

Reply:
i'd prefer program based on manpage and spec, instead of copying existing code. we can fix `signal_handler.cc` in a different commit.

Reply:
@tchaikov  thanks for your advice, I have modified it.